Bio/Phylo/_utils.py

```diff
@@ -82,6 +82,107 @@ def build_subgraph(graph, top):
     return G
 
 
+def to_igraph(tree, vertex_attributes=None, edge_attributes=("color", "width")):
+    """Convert a Tree object to an igraph Graph.
+
+    The result is useful for graph-oriented analysis and interactive plotting
+    with matplotlib, Cairo, and plotly.
+
+    Requires python-igraph version 0.10.0 or later.
+
+    :Parameters:
+        vertex_attributes : sequence of strings
+            A sequence of strings containing the Clade properties that are to
+            be stored as vertex attributes, e.g. "name".
+        edge_attributes : sequence of strings
+            A sequence of strings containing the Clade properties that are to
+            be stored as edge attributes, e.g. "color" and "width".
+
+    In both cases, if a property is not found it will be ignored.
+    """
+    try:
+        import igraph as ig
+    except ImportError:
+        raise MissingPythonDependencyError(
+            "Install igraph if you want to use to_igraph."
+        ) from None
+
+    if ig.__version__ < "0.10":
+        raise MissingPythonDependencyError(
+            "Update igraph to 0.10 or later if you want to use to_igraph."
+        ) from None
```

Comment:
`from None` is not needed here; we are not in an exception block.

Reply:
done